backend/app/views.py

```diff
@@ -22,10 +23,54 @@ def load_africa_geojson() -> dict:
         africa_geojson_string = africa_geojson_file.read()
     return json.loads(africa_geojson_string)
 
+
 @api_view(['GET'])
 def africa_map_geojson(request):
     """
     Load Africa map GeoJSON for frontend
     """
     africa_geojson = load_africa_geojson()
     return Response(africa_geojson)
+
+
+def load_democracy_data():
+    """
+    Read the CSV file of the democracy scores in Africa from disk
+    and return a parsed json array
+    """
+    filename = 'lieberman_afr_data.csv'
+    path = Path(settings.BACKEND_DATA_DIR, filename)
+    democracy_data = []
+    with open(path, encoding='utf-8') as democracy_score_file:
+        reader = csv.reader(democracy_score_file, delimiter=',')
+        headers = []
+        current_country_data = {"democracy_scores": {}}
+        for row in reader:
```

Comment:
A little too much magic number stuff happening in this file read. Let's discuss offline.